File: src/models/Task.ts
export default class Task {
	id: string
	name: string
	description: string
	tags: string[]
	columnId: string

	constructor(id: string, name: string, description: string, tags: string[], columnId: string) {
		this.id = id
		this.name = name
		this.description = description
		this.tags = tags
		this.columnId = columnId
	}
}

File: backend/Models/Task.cs
namespace backend.Models;

public class Task
{
    public long Id { get; set; }
    public string Name { get; set; } = "";
    public string Description { get; set; } = "";
    public IList<string> Tags { get; set; } = new List<string>();
    public long ColumnId { get; set; }
}